examples/tuning/README.txt

```diff
@@ -0,0 +1,2 @@
+Hyper parameters tuning use cases
+---------------------------------
```

Comment:
I would propose to add here the current version of the module tested (i got some  
- trouble with Pillow (version 7.0 does not have PILLOW_VERSION
- trouble with talos : 0.6.4 required (0.6.3 is not enough)

Reply:
This is the code I used to create a virtualenv (python3 )

python -m venv /volatile3/frouin/DL/ve_pynet
source /volatile3/frouin/DL/ve_pynet/bin/activate

python -m pip install --upgrade pip
pip install torch==1.3.1+cpu torchvision==0.4.2+cpu -f https://download.pytorch.org/whl/torch_stable.html
pip install tensorflow
#pip install talos
pip install git+https://github.com/autonomio/talos
pip install scikit-image
pip install tabulate
pip install progressbar

cd ~/gitsforked/pynet
python setup.py  develop